### PR TITLE
fix: Update description for hobby upgrade script

### PR DESCRIPTION
**Pull request by danielthedifficult**
## Problem

In the  original hobby-upgrade warning's language (written by yours truly), I don't mention which version number nor which dates are relevant to the warning.

## Changes

[x] Added a check to see if the containers exist or not (`docker volume ls | grep -Pzoq 'root_cl2ickhouse-data\n(.|\n)*root_postgres-data\n'`) 
[x] Added the dates and version number now allows users to better understand if the warning is relevant to them. 
[x] Minor improvements to the language and instructions

## How did you test this code?

Tested in my own self-hosted PH hobby instance. 
The container checks work when containers are present, and fail when they are not.

### CODE REVIEW

**Comment by danielthedifficult** [comment]
Converted to draft, I forgot I'd need to update the second part of the warning as well.

**Comment by danielthedifficult** [comment]
Now tracks warning states for better user prompts
Uses state from pre-update volume presence check to show post-install warning about data loss in addition to pre-upgrade warning.

Behavior:
If user has persistent volumes present, no warnings are shown.
If user is missing the persistent volumes, show warnings before upgrading and before restarting the stack.

**Comment by danielthedifficult** [comment]
@hazzadous Given our history with this file, I'm pinging you to request a review and/or approval. Cheers!

File: bin/upgrade-hobby [hunk 1]
```diff
@@ -11,31 +11,41 @@ else
...
-echo ""
-echo "✅ back up your entire environment/installation (vm, host, etc.), including all docker containers and volumes:"
-echo "✅ specifically back up the contents of :"
-echo "  ☑ /var/lib/postgresql/data in the postgres (root_db_1) container"
-echo "  ☑ /var/lib/clickhouse in the root_clickhouse_1 (root_clickhouse_1) container"
-echo "and be ready to check/recopy the data before you boot Posthog next."
-read -r -p "Do you want to upgarde PostHog? [y/N] " response
-if [[ "$response" =~ ^([yY][eE][sS]|[yY])+$ ]]
+echo "Checking for named postgres and clickhouse volumes to avoid data loss when upgrading from < 1.39"
+if docker volume ls | grep -Pzoq 'root_clickhouse-data\n(.|\n)*root_postgres-data\n'
```
Comment (hazzadous) [inline bin/upgrade-hobby:15]: hmm I have a feeling that the `root_` prepend is directory dependeny @danielthedifficult , am I wrong? I haven't checked. If not then all good.

**Review by fuziontech — APPROVED** [review]
Love it. Thanks @danielthedifficult !

**Pull request merged**